examples/amm/README.md

```diff
@@ -100,74 +100,118 @@ linera service --port $PORT &
 To properly setup the tokens in the proper chains, we need to do some transfer operations:
 
 - Transfer 50 FUN1 from `$OWNER_AMM` in `$CHAIN_AMM` to `$OWNER_1` in `$CHAIN_1`, so they're in the proper chain
+
 ```gql,uri=http://localhost:8080/chains/$CHAIN_AMM/applications/$FUN1_APP_ID
-    mutation {
-        transfer(
-            owner: "User:7136460f0c87ae46f966f898d494c4b40c4ae8c527f4d1c0b1fa0f7cff91d20f",
-            amount: "50.",
-            targetAccount: {
-                chainId: "dad01517c7a3c428ea903253a9e59964e8db06d323a9bd3f4c74d6366832bdbf",
-                owner: "User:65adbf65c9c1f48a0f1f2d06e0780994dcf8e428ffd5ee53948b3bb6c572c66f",
-            }
-        )
-    }
+mutation Transfer($ownerAmm: AccountOwner!, $chain1: ChainId!, $owner1: AccountOwner!, $amount: Amount!) {
+    transfer(
+        owner: $ownerAmm,
+        amount: $amount,
+        targetAccount: {
+            chainId: $chain1,
+            owner: $owner1,
+        }
+    )
+}
+
+variables-json:
```

Comment:
@andresilva91 ok but now it's not obvious anymore for a user what to do with this

Reply:
Reverted to more basic version